src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,7 @@
+package duke;
+
+public class DukeException extends Exception {
+    public DukeException() {
+        super("Sorry, duke.Duke cannot understand your command");
```

Comment:
I think more varied exceptions can be considered. Also, a suggestion: `duke.Duke` could be replaced with just `Duke`.